docs/index.md

```diff
@@ -61,6 +61,30 @@ An example that uses GFMD synatx for task lists:
   </td>
 </tr>
 </table>
+
+An example that uses KaTeX to generate math equations:
+
+
+<table>
+<tr>
+  <td>
+
+```markdown
+Euler's Identity \(e^{i\pi}+1=0\) is a beautiful formula.
+```
+  </td>
+  <td>&nbsp;→&nbsp;</td>
+  <td><br>
+<tip-box>
+
+Euler's Identity \(e^{i\pi}+1=0\) is a beautiful formula.
```

Comment:
A little curious whether an expression like `e^{{\frac{1}_{3}} + 1}` will cause nunjucks to misbehave in Markbind now. Do we need this: https://markbind.org/userGuide/tipsAndTricks.html#using-raw-endraw-to-display-content?

Reply:
![image](https://user-images.githubusercontent.com/38814428/92307208-35e5bb80-efc7-11ea-9b9c-d74253a85a06.png)
It does seems to be causing nunjucks rendering issues. I'm not too familiar with the intricacies nunjucks (like how to allow for `{{` here), perhaps @ang-zeyu can advise?

Reply:
we allow nunjucks syntax anywhere, even in seemingly unintuitive places like fenced code blocks (we even use this in our ug), the downside of this flexibility is that `{% raw %} {% endraw %}` will be necessary to resolve any ambiguity.

since this seems like a common use case (again correct me if im wrong, havent used latex much before), maybe you can add a warning `<box>` to the ug per @crphang's observation!

`v-pre` shouldn't be though, versus using a client side tex math solution like mathjax, since the equations are spit out at compile time

Reply:
When working on nunjuck-escaped special tags with @ang-zeyu, latex's syntax was one of the consideration. Maybe this can be wrapped in nunjuck-escaped special tags? You can check out puml plugin if it helps!

Reply:
> nunjuck-escaped special tags

@crphang this hasen't been implemented yet, I don't think we should either, theres several cases where its useful to have nunjucks syntax complement the "special tag", e.g. https://github.com/MarkBind/markbind/issues/1256

Reply:
wrapping texmath in html may also be rather troublesome (usage wise) to be worth the cost; that said if there is a way to use only the same `\[ \(` syntax that'd be great though (likely needs to be done via patching nunjucks itself)